zerver/webhooks/zabbix/doc.md

```diff
@@ -0,0 +1,87 @@
+Receive Zabbix notifications in Zulip!
+
+1.  {!create-stream.md!}
+
+2.  {!create-a-bot-indented.md!}
+
+    Construct the URL for the {{ integration_display_name }} bot using the bot's API key and the desired stream name:
+
+    {!webhook-url.md!}
+
+    Modify the parameters of the URL above, where `api_key` is the API key of your Zulip bot, and `stream`
+    is the stream name you want the notifications sent to. If you do not specify a `stream`,
+    the bot will send notifications to the `alerts` stream which you will need to create.
+
+3.  Create a script called `zulip.sh` in your `alertscripts` folder on your **Zabbix Server** containing
+    the below code. You can also find the file [here]("https://github.com/adambirds/zabbix-to-zulip-script").
+    The **alertscripts** folder is usually `/usr/lib/zabbix/alertscripts`.
+    Yours may be different depending on your environment. Make sure the script is executable by your
+    **Zabbix** environment.
+
+        #!/bin/bash
+
+        webhook_url="$1"
+
+        # Build our JSON payload and send it as a POST request to the Zulip incoming web-hook URL
+        payload="$3"
+        echo $payload > /tmp/payload.txt
+        /usr/bin/curl -m 5 --data "$payload" "${webhook_url}"
+
+4.  Go to Zabbix Web Interface and click **Administration** and then **Media Types**. Click **Create Media Type**.
+
+    Enter `Zulip` as the **name** of the media type. Select `Script` as the **type**.
+
+    Enter `zulip.sh` as the **Script name**. Enter `{ALERT.SENDTO}` as the **first** parameter.
+
+    Enter `{ALERT.SUBJECT}` as the **second** parameter. Enter `{ALERT.MESSAGE}` as the **third** parameter.
+
+    Tick **Enabled**. You should end up with the fields being filled out as shown below:
+
+    ![](/static/images/integrations/zabbix/001.png)
+
+    Click **Update**.
+
+5.  Go to Zabbix Web Interface and click **Administration** and then **Users**.
+    Click the **alias** of the user in which you will use to set the notification.
+
+    Once in the user's settings, click **Media**. Click **Add**. Set **Type** to `Zulip`.
+    Set **Send To** as the URL you constructed in step 2.
+
+    Tick the severity's in which in you wish to notify for and tick **Enabled**.
+    You should end up with the fields being filled similar to below:
+
+    ![](/static/images/integrations/zabbix/002.png)
```

Comment:
I don't think we need any of these screenshots (except the last one showing the example message). :)

Reply:
I would recommend removing all the screenshots. We generally try to not have any screenshots unless they are absolutely necessary. Please see the [integration docs writing guide](https://chat.zulip.org/api/integration-docs-guide) for more information! :)